rubric/rubric.json

```
@@ -1,80 +1,94 @@
-      "Expert": "Able to discuss technology with the broadest range of stakeholders from the form of simplified executive presentations to in-depth debates with solution architects on the details of tool selection. Passively builds knowledge and insight in more junior team members.\r"
+      "Beginner": "I am still learning technical vocabulary. I can explain a concept if I have seen it or read about it and have also implemented it several times.",
+      "Advanced Beginner": "I can communicate clearly with higher level developers but still struggles to discuss things with non-technical stakeholders. I can help beginners with obvious problems but will defer fairly frequently to more senior developers.",
+      "Intermediate": "I am becoming more comfortable discussing technical topics with non-technical stakeholders. I can fluently discuss topics with correct vocabulary. I am able to identify patterns in the system and constantly search for ways to simplify it but can occasionally miss details.",
+      "Advanced": "I am capable of fluently discussing technical topics with simplified and direct vocabulary. I can effectively communicate with others through both verbal and written communication and able to match my descriptions to the intended audience.",
+      "Expert": "I am able to discuss technology with the broadest range of stakeholders from executive presentations to in-depth debates with solution architects on the details of tool selection. I enjoy building knowledge and insight in more junior team members."
     }
   },
   "Core Software Skills": {
-    "Data Structures": {
-      "Beginner": "Understands simple control flow and some basic collections. May fit incorrect but familiar data structures to far too many problems.",
-      "Advanced Beginner": "Understands most control flow and collections. Can sometimes tell when an algorithm can be improved but probably won't know how. Follows established patterns but cannot chart a new course.",
-      "Intermediate": "Can optimize advanced beginner code call out reasons for performance deficiencies and understands the basics of how data access and networking calls can affect performance.",
-      "Advanced": "Strong understanding of data structures and rarely makes degenerative code mistakes. Intimately understands data access and network calls. Knows hot spots where technical debt lives and must be addressed at a solution level.",
-      "Expert": "Can implement high-tier algorithms with maximum performance. Understands how to build rube-goldberg/code golf structures - and uses this knowledge to avoid them at all costs. Strong understanding of popular and valuable design patterns.\r"
+    "General": {
+      "Beginner": "I can complete assigned tasks under supervision. I am ramping up on core engineering concepts and languages. I follow prescribed processes and identify when to seek guidance.",
```

```suggestion
      "Beginner": "I can complete assigned tasks under supervision. I am learning core engineering concepts and languages. I follow prescribed processes and may have trouble identifying when to seek guidance.",
```

Avoid buzzwords. Knowing when to seek guidance is something that a beginner may not posses.